onnx/defs/traditionalml/defs.cc

```diff
@@ -915,131 +946,195 @@ ONNX_ML_OPERATOR_SET_SCHEMA(
+            AttributeProto::INTS,
+            true)
         .Attr(
-            "aggregate_function",
-            "Defines how to aggregate leaf values within a target. <br>One of 'AVERAGE,' 'SUM,' 'MIN,' 'MAX.'",
-            AttributeProto::STRING,
-            std::string("SUM"))
+            "nodes_trueleafs",
+            "1 if true branch is leaf for each node and 0 an interior node. To represent a tree that is a leaf (only "
+            "has one node), one can do so by having a single `nodes_*` entry with true and false branches referencing "
+            "the same `leaf_*` entry",
+            AttributeProto::INTS,
+            true)
         .Attr(
-            "base_values",
-            "Base values for regression, added to final prediction after applying aggregate_function; the size must be the same as the classes or can be left unassigned (assumed 0)",
-            AttributeProto::FLOATS,
+            "nodes_falseleafs",
+            "1 if false branch is leaf for each node and 0 if an interior node. To represent a tree that is a leaf "
+            "(only has one node), one can do so by having a single `nodes_*` entry with true and false branches "
+            "referencing the same `leaf_*` entry",
+            AttributeProto::INTS,
+            true)
+        .Attr(
+            "nodes_missing_value_tracks_true",
+            "For each node, define whether to follow the true branch (if attribute value is 1) or false branch (if "
+            "attribute value is 0) in the presence of a NaN input feature. This attribute may be left undefined and "
+            "the default value is false (0) for all nodes.",
+            AttributeProto::INTS,
             OPTIONAL_VALUE)
         .Attr(
-            "base_values_as_tensor",
-            "Base values for regression, added to final prediction after applying aggregate_function; the size must be the same as the classes or can be left unassigned (assumed 0)",
+            "tree_roots",
+            "Index into `nodes_*` for the root of each tree. The tree structure is derived from the branching of each "
+            "node.",
+            AttributeProto::INTS,
+            true)
+        .Attr(
+            "membership_values",
+            "Members to test membership of for each set membership node. List all of the members to test again in the "
+            "order that the 'SET_MEMBER' mode appears in `node_modes`, delimited by `NaN`s. Will have the same number "
+            "of sets of values as nodes with mode 'SET_MEMBER'. This may be ommitted if the node doesn't contain any "
+            "'SET_MEMBER' nodes.",
             AttributeProto::TENSOR,
             OPTIONAL_VALUE)
+        .Attr(
+            "leaf_targetids",
+            "The index of the target that this leaf contributes to (this must be in range `[0, n_targets)`).",
+            AttributeProto::INTS,
+            true)
+        .Attr("leaf_weights", "The weight for each target", AttributeProto::TENSOR, true)
```

Comment:
Should we change the name?

Reply:
I think we should change the description to be `"The weight for each leaf"` instead given the `leaf_*` attributes define a table.